tests/bootstrap.php

```diff
@@ -4,9 +4,7 @@
 
 require dirname(__DIR__).'/vendor/autoload.php';
 
-if (method_exists(Dotenv::class, 'bootEnv')) {
```

Comment:
This is right out of the blueprint. Worries me to remove it, but I think I'm ok with it.